src/main/java/cart/auth/BasicAuthorizationExtractor.java

```diff
@@ -0,0 +1,35 @@
+package cart.auth;
+
+import cart.auth.credential.CredentialEntity;
+import javax.servlet.http.HttpServletRequest;
+import org.apache.tomcat.util.codec.binary.Base64;
+import org.springframework.stereotype.Component;
+
+@Component
+public class BasicAuthorizationExtractor {
+
+    private static final String AUTHORIZATION = "Authorization";
+    private static final String BASIC_TYPE = "Basic";
+    private static final String DELIMITER = ":";
+
+    public CredentialEntity extract(HttpServletRequest request) {
+        String header = request.getHeader(AUTHORIZATION);
```

Comment:
HttpHeaders.AUTHORIZATION 을 활용 할수 있겠네요
```suggestion
        String header = request.getHeader(HttpHeaders.AUTHORIZATION);
```

Reply:
수정하였습니다